src/main/java/uk/ac/sanger/sccp/stan/service/releasefile/ReleaseColumn.java

```diff
@@ -70,6 +71,10 @@ public String get(ReleaseEntry entry) {
 
     @Override
     public String toString() {
+        if (this.name() == "Visium_concentration") {
+            // A bit hacky but because we are using an invalid char we can't include it by default in the enum name
```

Comment:
@khelwood Bit of a weird one here, would be good to get your thoughts on two things: 
- One thing I did was use 'u' instead of 'μ' over parsing concerns. While testing this, excel mis-interpreted the character as something random so thought 'u' was a safer bet.
- Not sure if there is a better way to do this if check. Some things I thought about were: changing enum name to 'Visium_concentration_pg_ul' so we dont need this hacky bit (doesnt look as good) or changing the structure of each enum so that they have a 'headerName' attribute which is a string passed in for the column header which allows illegal chars.

Reply:
We had some issue before (x833) with Excel mispresenting non-ascii characters. Excel has a weird default encoding. Using `u` instead seems reasonable: I don't think the scientists care.

Reply:
Ok makes sense, thanks.